### ISSUE TITLE
Better app icon

### ISSUE THREAD

**Issue by florianbuerger** [florianbuerger/brisk-ios#16]
The current app icon is just a placeholder

**Comment by ethanhuang13** [comment]
I tried to use Mac app icon. But it's probably too complex to downsize for iOS icon. 

Could we contact the author(Samantha Broccoli) for help?  @ZevEisenberg

![8c4a7458-6220-4028-999b-2633fe623c67](https://user-images.githubusercontent.com/12409307/42943415-48297f5e-8b95-11e8-9857-48d56ddcea8b.png)

**Comment by florianbuerger** [comment]
Yeah, I know. Verena (@verena-von-o) will give it try, let's see where we end up.

**Comment by ZevEisenberg** [comment]
Samantha is looking into it, but if @Verena-von-O is available to try as well, that'll just give us more options to choose from.

**Comment by Verena-von-O** [comment]
Hi, let me know if Samantha doesn't have time, then I will look into it.

**Comment by florianbuerger** [comment]
@ZevEisenberg any news from Samantha? No problem if not, I am just curious.

In the meantime, I've updated the placeholder thanks to http://www.parakeet.co/primaries/ 
![simulator screen shot - iphone se - 2018-08-12 at 17 06 09](https://user-images.githubusercontent.com/119219/44006429-47fa13b6-9e52-11e8-8855-ba6099a4e09b.png)

**Comment by ZevEisenberg** [comment]
She said she doesn’t have one after all.
